androidsdk/src/test/java/com/wootric/androidsdk/objects/WootricCustomThankYouTest.java

```diff
@@ -9,81 +9,217 @@
+        assertThat(customThankYou.getTextForScore(7, "CES")).isEqualTo("thank you");
+    }
+
+    @Test
+    public void returnsDetractorLinkText_forCESDetractorScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setDetractorLinkText("detractor");
+
+        assertThat(customThankYou.getLinkTextForScore(1, "CES")).isEqualTo("detractor");
+        assertThat(customThankYou.getLinkTextForScore(3, "CES")).isEqualTo("detractor");
+    }
+
+    @Test
+    public void returnsPassiveLinkText_forCESPassiveScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setPassiveLinkText("passive");
+
+        assertThat(customThankYou.getLinkTextForScore(4, "CES")).isEqualTo("passive");
+        assertThat(customThankYou.getLinkTextForScore(5, "CES")).isEqualTo("passive");
+    }
+
+    @Test
+    public void returnsPromoterLinkText_forCESPromoterScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setPromoterLinkText("promoter");
+
+        assertThat(customThankYou.getLinkTextForScore(6, "CES")).isEqualTo("promoter");
+        assertThat(customThankYou.getLinkTextForScore(7, "CES")).isEqualTo("promoter");
+    }
+
+    @Test
+    public void returnsDefaultLinkText_CES() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setLinkText("thank you");
+
+        assertThat(customThankYou.getLinkTextForScore(1, "CES")).isEqualTo("thank you");
+        assertThat(customThankYou.getLinkTextForScore(7, "CES")).isEqualTo("thank you");
+    }
+
+    @Test
+    public void returnsDetractorQuestion_forCSATDetractorScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setDetractorText("detractor");
+
+        assertThat(customThankYou.getTextForScore(1, "CSAT")).isEqualTo("detractor");
+        assertThat(customThankYou.getTextForScore(2, "CSAT")).isEqualTo("detractor");
+    }
+
+    @Test
+    public void returnsPassiveQuestion_forCSATPassiveScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setPassiveText("passive");
+
+        assertThat(customThankYou.getTextForScore(3, "CSAT")).isEqualTo("passive");
+    }
+
+    @Test
+    public void returnsPromoterQuestion_forCSATPromoterScore() throws Exception {
+        WootricCustomThankYou customThankYou = new WootricCustomThankYou();
+        customThankYou.setPromoterText("promoter");
+
+        assertThat(customThankYou.getTextForScore(4, "CSAT")).isEqualTo("promoter");
+        assertThat(customThankYou.getTextForScore(5, "CSAT")).isEqualTo("promoter");
+    }
+
+    @Test
+    public void returnsDefaultQuestion_forCSAT() throws Exception {
```

Comment:
pick `_` or snakeCase but don't mix them, I believe this applies to all specs for CES/CSATt